cachix-0.1.0.0 build failure with fsnotify-0.3

As seen on the Stackage build server:

```
src/Cachix/Client/Commands.hs:165:13: error:
    • The constructor ‘Removed’ should have 3 arguments, but has been given 2
    • In the pattern: Removed fp _
      In an equation for ‘action’:
          action (Removed fp _)
            = pushStorePath env config name $ toS $ dropEnd 5 fp
      In an equation for ‘push’:
          push env config name _ True
            = withManager
                $ \ mgr
                    -> do _ <- watchDir mgr "/nix/store" filterF action
                          ....
            where
                action :: Action
                action (Removed fp _)
                  = pushStorePath env config name $ toS $ dropEnd 5 fp
                action _ = return ()
                filterF :: ActionPredicate
                filterF (Removed fp _) | ".lock" `isSuffixOf` fp = True
                filterF _ = False
                ....
    |
165 |     action (Removed fp _) = pushStorePath env config name $ toS $ dropEnd 5 fp
    |             ^^^^^^^^^^^^

src/Cachix/Client/Commands.hs:169:14: error:
    • The constructor ‘Removed’ should have 3 arguments, but has been given 2
    • In the pattern: Removed fp _
      In an equation for ‘filterF’:
          filterF (Removed fp _) | ".lock" `isSuffixOf` fp = True
      In an equation for ‘push’:
          push env config name _ True
            = withManager
                $ \ mgr
                    -> do _ <- watchDir mgr "/nix/store" filterF action
                          ....
            where
                action :: Action
                action (Removed fp _)
                  = pushStorePath env config name $ toS $ dropEnd 5 fp
                action _ = return ()
                filterF :: ActionPredicate
                filterF (Removed fp _) | ".lock" `isSuffixOf` fp = True
                filterF _ = False
                ....
    |
169 |     filterF (Removed fp _) | ".lock" `isSuffixOf` fp = True
    |              ^^^^^^^^^^^^
```

--- a/cachix/ChangeLog.md
+++ b/cachix/ChangeLog.md
@@ -7,9 +7,17 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
-- #86 Support fsnotify 0.3.x @domenkozar
+## [0.1.0.1] - 2018-07-05
+
+### Changed
+
+- #92 Add build-tool-depends where needed @domenkozar
+- #90 HLint fixes @domenkozar
+- #86 Improve Cabal description and synopsis @domenkozar
+- #87 Support fsnotify 0.3.x @domenkozar
 
 ## [0.1.0.0] - 2018-07-01
 
 ### Added
+
 - Initial release @domenkozar

